packages/galileo-cli/src/commands/document/flags.ts

```diff
@@ -14,6 +14,7 @@ export interface DocumentUploadCommandFlags extends BaseDocumentCommandFlags {
   bucketName?: string;
   uploadKeyPrefix?: string;
   metadataFile?: string;
+  language?: string;
```

Comment:
let's keep the `modelRefKey` name or simplified version `modelRef`

Reply:
Resolved in v2.